app/screens/search/search_result_post/search_result_post.test.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {shallowWithIntl} from 'test/intl-test-helper';
+
+import SearchResultPost from './search_result_post';
+
+describe('SearchResultPost', () => {
+    const baseProps = {
+        goToThread: jest.fn(),
+        onHashtagPress: jest.fn(),
+        onPermalinkPress: jest.fn(),
+        previewPost: jest.fn(),
+        postId: 'post-id',
+        isDeleted: false,
+        highlightPinnedOrFlagged: false,
+        managedConfig: {},
+        showFullDate: false,
+        skipFlaggedHeader: false,
+        skipPinnedHeader: false,
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = shallowWithIntl(<SearchResultPost {...baseProps}/>);
+
+        expect(wrapper.instance()).toMatchSnapshot();
```

Comment:
I'm good as is but would be nice to have other tests on top of snapshot (ex. checking of state for a given props/action, etc.) rather than a snapshot alone. For snapshot, please use `.getElement` to output wrapped element instead of class instance.
```suggestion
        expect(wrapper.getElement()).toMatchSnapshot();
```

Also, we already have `@testing-library/react-native` in place and I'd suggest to start using it in the future as much as possible.

Reply:
Updated